<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportMulti</class>
 <widget class="QWidget" name="ExportMulti">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>437</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Export Location</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Filename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="FileName">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Export location</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLineEdit" name="ExportLocation"/>
        </item>
        <item>
         <widget class="QPushButton" name="BrowseButton">
          <property name="text">
           <string>Browse</string>
          </property>
          <property name="icon">
           <iconset resource="../images.qrc">
            <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Export Format</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="HTML">
        <property name="text">
         <string>&amp;HTML (preserves formatting)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="PlainText">
        <property name="text">
         <string>&amp;Plain text (discards formatting)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Export Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="IncludeExportDate">
        <property name="text">
         <string>Include &amp;export date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ExportLooseFiles">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, RoboJournal exports the entire journal as a series of loose individual entries instead of merging everything into one large file (as is the default behavior). &lt;/p&gt;&lt;p&gt;This method automatically creates a new sub-folder for this purpose in the current &lt;span style=&quot; font-weight:600;&quot;&gt;Export location&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>&amp;Keep individual entries separate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PermitName">
        <property name="text">
         <string>Allow custom file&amp;name</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="SortAscending">
        <property name="text">
         <string>Sort in &amp;ascending order (oldest entries first)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="SortDescending">
        <property name="text">
         <string>Sort in &amp;descending order (latest entries first)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
